tests/ostree-spawn.c

```diff
@@ -135,11 +168,20 @@ ostree_commit (GFile *repo,
       { "gpg-homedir", gpg_home_path },
       { "timestamp", formatted_timestamp },
       { NULL, raw_tree_path },
-      { NULL, NULL }
     };
+  CmdArg empty = { NULL, NULL };
+  g_autoptr(GArray) cmd_args = g_array_sized_new (FALSE, TRUE, sizeof (CmdArg), 8);
+  g_autoptr(GPtrArray) formatted_cmd_args_membuf = g_ptr_array_new_with_free_func (g_free);
+  memcpy (cmd_args->data, initial_args, sizeof(initial_args));
+  cmd_args->len = G_N_ELEMENTS (initial_args);
```

Comment:
This should probably be `g_array_append_vals()`, right?

Reply:
That could also work.